src/store/cart.module.js

```diff
@@ -33,16 +33,14 @@ const actions = {
     context.commit(PURGE_CART)
   },
   [SET_ORDER] (context, input) {
-    const itemIds = Object.values(input.cart).reduce((array, item) => {
-      array.push(item.id)
-      return array
-    }, [])
-    const orderParams = {
-      items: itemIds
-    }
-    console.log(itemIds)
+    console.log(input)
+    const items = Object.values(input.cart).reduce((hash, item) => {
+      hash[item.id] = { amount: item.amount }
```

Comment:
This had to be updated to pass the amount along with the id so that the rails backend could calculate the total for the order efficiently